public/util/numberToString.js

```diff
@@ -1,11 +1,10 @@
-import can from "can";
-import "can/view/stache/";
+import stache from "can-stache";
 
-can.stache.registerHelper("numberToString", function(newVal, source){
-	if(newVal instanceof can.expression.SetIdentifier) {
-		source(newVal.value === "" ? null : +newVal.value);
-	} else {
-		source = newVal;
-		return source() + "";
-	}
-});
\ No newline at end of file
+stache.registerConverter("numberToString",{
```

Comment:
Do you know where can I find documentation for the converter? Can't seem to find anything on the http://v3.canjs.com/doc site.